pkg/autoscaler/scaling/autoscaler.go

```diff
@@ -247,7 +247,7 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) ScaleResult {
 		// numAct stays 1, only needed to scale from 0.
 	case a.deciderSpec.TargetBurstCapacity > 0:
 		totCap := float64(originalReadyPodsCount) * a.deciderSpec.TotalValue
-		excessBCF = math.Floor(totCap - observedStableValue -
+		excessBCF = math.Floor(totCap - observedPanicValue -
```

Comment:
Comment above needs to be updated as well. It still says using stable value.

Reply:
Done.